npm package not being uploaded

```
NPM API key format changed recently. If your deployment fails, check your API key in ~/.npmrc.
http://docs.travis-ci.com/user/deployment/npm/
npm ERR! need auth auth and email required for publishing
npm ERR! need auth You need to authorize this machine using `npm adduser`
npm ERR! System Linux 3.13.0-40-generic
npm ERR! command "/home/travis/.nvm/v0.10.36/bin/node" "/home/travis/.nvm/v0.10.36/bin/npm" "publish"
npm ERR! cwd /home/travis/build/mozilla/payments-config
npm ERR! node -v v0.10.36
npm ERR! npm -v 1.4.28
npm ERR! code ENEEDAUTH
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /home/travis/build/mozilla/payments-config/npm-debug.log
npm ERR! not ok code 0
```

I tried to make a new API key in the command line, but that new key didn't seem to work for Travis. I did 0.0.11 manually which is yucky, especially because to prove why I shouldn't do that, I accidentally published (then had to unpublish) 0.0.12. Ugh.

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,6 +1,8 @@
 language: python
 python:
 - '2.7'
+before_install:
+  npm install -g npm@'>=2.13.5'
 install:
 - pip install flake8 twine
 - python setup.py develop

